pr-1.js

```diff
@@ -0,0 +1,25 @@
+window.currentUser = { id: '19', name: 'Jane', email: 'jane@chameleon.io' };
+
+// users are like authors and profiles like commentors
+// open a modal with commentor info when clicked
+
+
+export const  ActiveProfiles({ profiles, onLaunchProfile }) => {
+  var active = [];
+
+  for(i=0; i < profiles.length; i++) {
+    if(!profiles[i].disabled && profiles[i]['last_seen_time'] > new Date(new Date().getTime()-(24*60*1000)).toISOString()) { // within the last 24 hours
+      active.push(profiles[i]);
+    }
+  }
+  
+  if(active.length == 1 && active[0].email  === window.currentUser.email) {
+    active.length = 0;
+  }
+
+  return (
+    <div>
+       {active.map(function(a) { return <div onClick={() => onLaunchProfile(a.name, a.email)}>{a.name} - {a.email}</div> })}
+    </div>
+  )
```

Comment:
Some changes were made here:
- Function declaration to Arrow function:
   -  Why: The arrow function syntax (() =>) is more concise and readable, making the code more modern and aligned with current JavaScript practices.
- Added a `key` prop
  - Why: Adding the key prop to each rendered element within a loop, such as map, is a recommended practice by React to optimize rendering and uniquely identify elements.
- Renamed a prop to profile
  - Why: To be clear

```suggestion
return (
  <div>
    {filteredProfiles.map((profile) => (
      <div
        key={profile.id}
        onClick={() => onLaunchProfile(profile.name, profile.email)}
      >
        {profile.name} - {profile.email}
      </div>
    ))}
  </div>
);

```